angular-workspace/projects/ni/nimble-angular/pipes/format-number-text.pipe.ts

```diff
@@ -0,0 +1,25 @@
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
+import { NumberTextUnitFormat, NumberTextUnitFormatOptions } from '@ni/nimble-components/dist/esm/table-column/number-text/models/number-text-unit-format';
+
+/**
+ * A pipe that transforms a number into a string with optional unit
+ */
+@Pipe({
+    name: 'formatNumberText',
+    standalone: true
+})
+export class FormatNumberTextPipe implements PipeTransform {
```

Comment:
Comparing to other angular pipe names: https://angular.io/guide/pipes
| date
| number
| percent

They don't seem to include an action (i.e. `format`, I guess because they all do the format action since that's what all pipes do), maybe omit the action from the name like:
- `numberText` for the name
- `NumberTextPipe` for the class